Meta/run.py

```diff
@@ -484,7 +484,8 @@ def set_up_disk_image_path(config: Configuration):
 
 def set_up_cpu(config: Configuration):
     if config.qemu_kind == QEMUKind.NativeWindows:
-        config.qemu_cpu = "max,vmx=off"
+        # Setting -cpu to "max" breaks on QEMU 9 with WHPX whereas the default cpu is more broadly compatible
+        config.qemu_cpu = None
```

Comment:
Can you try using `host` or `host,vmx=off` as well?

Reply:
This was already answered on discord: The `host` CPU type doesn't seem to be available on windows and it would likely break arm windows systems.